src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -100,6 +109,17 @@ public HostStatus GetHostStatus()
             return status;
         }
 
+        [HttpPost]
+        [Route("admin/host/debug")]
+        public bool LaunchDebugger()
+        {
+            if (_webHostSettings.IsSelfHost)
+            {
+                return Debugger.Launch();
```

Comment:
it does that internally (https://referencesource.microsoft.com/#mscorlib/system/diagnostics/debugger.cs,94)